### ISSUE TITLE
Improve orderlist key & slot

### ISSUE THREAD

**Issue by ngtuna** [BuildOnViction/tomochain-v1#600]
https://github.com/tomochain/tomochain/blob/tomoX/tomox/orderlist.go#L68

**Current**

Key (in RBT & DB) = big.Int(OT.slot + price) -> to BigToHash.Bytes()
Slot:
```go
if orderTree.orderBook != nil {
		orderList.slot = orderTree.orderBook.Slot
} else {
	orderList.slot = new(big.Int).SetBytes(crypto.Keccak256(key))
}
```

**Expect**
Key (in RBT) = price
Key (in DB) = []byte(pairName) + key (in RBT)
Slot = key.ToBigInt

**Comment by ngtuna** [comment]
After carefully investigating, we decided to keep Key in both RBT and DB are the same that is `[]byte(pairName) + key (in RBT)` as []byte(pairName) converted to bigInt added same amount to all nodes so that it doesn't affect the comparator function of RBT

**Comment by ngtuna** [comment]
Fixed at #602
